gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/resource/environment/EnvironmentAnalyticsResourceTest.java

```diff
@@ -62,6 +66,9 @@ class EnvironmentAnalyticsResourceTest extends AbstractResourceTest {
     @Inject
     private final ApiQueryServiceInMemory apiQueryService = new ApiQueryServiceInMemory();
 
+    @Inject
+    private final ApplicationQueryServiceInMemory applicationQueryService = new ApplicationQueryServiceInMemory();
```

Comment:
```suggestion
    private final ApplicationQueryServiceInMemory applicationQueryService;
```

It’s useless to initialise with `@Inject`.

But we need to reset the object in `tearDown()`